alchemist-dsl-processor/build.gradle.kts

```diff
@@ -0,0 +1,25 @@
+import Libs.alchemist
+
+plugins {
+    id("kotlin-multiplatform-convention")
+}
+kotlin {
+    jvm()
+    sourceSets {
+        val jvmMain by getting {
+            dependencies {
+                api(alchemist("api"))
+                implementation(libs.ksp.api)
+            }
+            kotlin.srcDir("src/main/kotlin")
+            resources.srcDir("src/main/resources")
```

Comment:
can you remove these and move sources to `src/commonMain/kotlin`? Same for the other sourcesets